epub33/rs/index.html

```diff
@@ -303,15 +304,19 @@ <h4>Parsing Relative URLs</h4>
 			<section id="sec-pkg-doc-base-dir">
 				<h4>Base Direction</h4>
 
-				<p id="confreq-rs-pkg-dir" data-tests="confreq-rs-pkg-dir.epub">If the <code>dir</code> attribute is set
-					and indicates a base direction of <code>ltr</code> or <code>rtl</code>, Reading Systems MUST
-					override the bidi algorithm per <a data-cite="bidi#Higher-Level_Protocols">the higher-level
-						protocols</a> defined in [[BIDI]], setting the paragraph embedding level to 0 if the base
-					direction is <code>ltr</code>, or 1 if the base direction is <code>rtl</code>.</p>
-
-				<p id="confreq-rs-pkg-dir-auto" data-tests="confreq-rs-pkg-dir-auto.epub">Otherwise the base direction
-					is <code>auto</code>, in which case Reading Systems MUST determine the text's direction by applying
-					the Unicode Bidi Algorithm, beginning with <a data-cite="bidi#P2">Rule P2</a> of [[BIDI]].</p>
+				<p id="confreq-rs-pkg-dir"
+				    data-tests="#confreq-rs-pkg-dir_rtl-001,#confreq-rs-pkg-dir_rtl-004,#confreq-rs-pkg-dir_rtl-005,#confreq-rs-pkg-dir_rtl-006">
```

Comment:
@iherman here and below on line 316, have I corrected interpreted which of your dir tests corresponds to which statement? (Please remember to update the spec with these references after you commit tests.)

Reply:
Ehem. I have submitted https://github.com/w3c/epub-specs/pull/1821 which waits for your approval :-)

Reply:
Oh no! I'm the worst. Sorry I missed that — epub-specs is so busy that I almost never check. I will need to figure out a better email filtering system so that I actually see the ones meant for me.